Automattic/wp-calypso
# Stats: Remove showNewModules and apply horizontal bars to other components

#### Proposed Changes

* applying horizontal bars to all pages needing it besides Traffic (mainly summary detail pages - visible when clicking on "View details/all")
* adding a feature flag to control the above
* adding a temporary header to the summary details page (behind a feature flag) to be updated in a following pull request
* extending Horizontal bar component to accommodate `Tags` that use an array for labels instead of a string (work in progress while the design is finalised)

This change focuses on removing `showNewModules` introduced with the horizontal bar component.
The `Insights` page and `summary detail` page are a work in progress, hidden behind a feature flag to be updated incrementally.

#### Testing Instructions



* go to the live branch
* without applying a feature flag, nothing should change visually anywhere (potential areas of focus - Traffic, Insights and summary details pages)
* apply the feature flag by adding `?flags=stats/horizontal-bars-everywhere` to the URL
* verify that Traffic, Insights and summary details pages are not broken but keep in mind that the last two places are under development

#### Pre-merge Checklist





Related to #71638
